drafts/Stormless.md

```diff
@@ -0,0 +1,198 @@
+
+Now let's run this locally.
+
+```
+sls stackstorm docker run --function InviteSlack --verbose --data '{"body": {"first_name": "Santa", 
+"email": "santa@mad.russian.xmas.com"}}'
+```
+
+Local runs in the container. It takes a bit longer, but ensures that the execution environment matches AWS lambda very closely, so better safe than sorry. 
+
+If I want to 
+
+Now we are really ready ready. Deploy the function to AWS, and run it "serverless".
+
+```
+sls deploy
+```
+
+It will take some while - now it's serverless (and honestly, our bundle is a bit bloated, patience! plugin developers are currently busy solving other problems, we will optimize it as soon as we can)
+
+
+> PRO TIP: if something goes wrong at this point, most likely your AWS setup is fucked up. Go to Getting Ready, step 2. Google, Stack-overflow.
+ 
+
+You might be curious to see how it looks in the AWS console. If the PRO in you is saying "no, you should stay cool and use CLI", just don't listen. Indulge yourself, open a browser and take a good look to your Lambda. While there, you might also inspect the API Gateway endpoint that `sls` created for you. 
+
+But to test it, we'll go back to terminal. Here is how to run it with `sls`:
+
+```
+sls invoke --function InviteSlack --log --data '{"body": {"first_name": "Santa", "email": "santa@mad.russian.xmas.com"}}'
+```
+
+Finally, let's POST to the API endpoint. The endpoint was printed at the end of `sls deploy` and you should have taken notice, but it's OK if you didn't: you can always get it by typing `sls info`. 
+
+You `curl` lovers go ahead use it to POST; be sure to set `Content-Type=application/json` header. Me - I'll show off with [httpie, aka CURL for humans](https://httpie.org/):
+
+```
+# DON'T copy-paste! Use YOUR endpoint!
+
+http --json POST  https://YOUR-ENDPOINT.amazonaws.com/dev/invite \
+email=test@example.com first_name=Dmitri
+```
+
+How did it go? Everything worked, at least for me. Let's fire yet another most useful `sls` command to check the CloudWatch logs:
+
+```
+sls logs --function InviteSlack
+```
+
+Success! And the end of the first episode. Enough for now: Christmas time is here, take it slow.
+
+The code example so far is on Github at [1-add-first-action](https://github.com/dzimine/slack-signup-serverless-stormless/tree/DZ/1-add-first-action). Grab `.gitignore` from there if you are using Git. Yes, you should. And if you're following alone, it's a good time to commit now, before we move to next step. Grab this `[.gitignore](https://github.com/dzimine/slack-signup-serverless-stormless/blob/DZ/1-add-first-action/.gitignore)` if you are using Git.
```

Comment:
Don't put this `.gitignore` in backticks, if you want it to be handled like a link. Is showing up broken in the GH render.

Reply:
👍